src/app/business/routes/model/components/detail/components/tabs/index.js

```diff
@@ -35,6 +35,24 @@ class ModelTabs extends Component {
         const {item, addNotification} = this.props;
         const {tabIndex} = this.state;
 
+        // do not display the "type" key in the traintuple that's been inserted for the fronten's internal use
+        const cleanTraintuple = item && item.traintuple && {...item.traintuple};
+        if (cleanTraintuple) {
+            delete cleanTraintuple.type;
```

Comment:
The selectors inject into each model a `traintuple` that has for content either the model's traintuple, compositeTraintuple or aggregatetuple with the addition of a `type` property. This makes the manipulation of models much, much easier throughout the code (and it was a mechanism that was originally created to handle bundles of models).

I just wanted to avoid displaying this type to the user because it's not the raw response from the API and I felt this was a quick and easy fix.

I could move the code display + type removal to a separate component though. This would make sure the removal is done only when the item changes and not when the full tabs component changes.